index.html

```diff
@@ -3494,6 +3493,15 @@ <h4 id="Types of Attack Roll">Types of Attack Roll</h4>
                   </li>
                </ul>
             </li>
+            <li><strong>Ability attack.</strong>
+               <ul>
+                  <li>An attack is considered an ability attack if it is made as part of an ability with limited uses, 
+                     such as using the Sweeping Strike or Ice Spikes abilities. Ability attacks are special because 
+                     even if you roll lower than the AC of the target, you still hit and deal glancing damage instead 
+                     (deal half damage). See the glancing damage section for more details.
```

Comment:
Not against this, but this will be a large buff to all attack abilities.  Resistable non-attack abilities (e.g. fireball, elemental grapple) will look significantly worse in comparison post this